portal/src/graphql/portal/AuthenticationAuthenticatorSettings.tsx

```diff
@@ -185,22 +186,44 @@ const AuthenticationAuthenticatorSettings: React.FC<Props> = function Authentica
     onSecondaryActivateClicked
   );
 
+  const onSaveButtonClicked = React.useCallback(() => {
+    console.log("save");
+    console.log(primaryAuthenticatorState, secondaryAuthenticatorState);
```

Comment:
Remove logs.

Please turn on [no-console](https://eslint.org/docs/rules/no-console) to prevent this.

Reply:
these 2 lines should be removed in later commit, will add lint rules to enforce no console.log